qemu/tests/sev_dhcert_boot.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from avocado.utils import process
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_package
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sev basic test on Milan and above host:
+    1. Check host sev capability
+    2. Generate dhcert and session files
+    3. Boot sev VM with dhcert
+    4. Verify sev enabled
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sev test", test.log.info)
+    timeout = params.get_numeric("login_timeout", 240)
+
+    output = process.getoutput(params["sev_host_check"], shell=True)
+    if output not in params["module_status"]:
+        test.cancel("Host sev capability check fail.")
+
+    sev_tool_pkg = params.get("sev_tool_pkg")
+    s, o = process.getstatusoutput("rpm -qa | grep %s" % sev_tool_pkg,
+                                   shell=True)
+    if s != 0:
+        install_status = utils_package.package_install(sev_tool_pkg)
+        if not install_status:
+            test.cancel("Failed to install %s." % sev_tool_pkg)
+
+    vm_name = params["main_vm"]
+    try:
+        process.system_output("sevctl export --full vm.chain", shell=True)
+        process.system_output("sevctl session --name " + vm_name +
```

Comment:
we'd better remove vm.chain and godh/session related files(in finally)

Reply:
`
files_remove = []
try:
    process.system_output("sevctl export --full vm.chain", shell=True)
    files_remove.append('vm.chain')
    process.system_output("sevctl session --name " + vm_name ....)
    files_remove.extend([f'{vm_name}_{name}' for name in ['godh.b64', 'session.b64', 'tek.bin', 'tik.bin']])
`